CYCLONEDDS_QUALITY_DECLARATION.md

```diff
@@ -14,65 +14,90 @@ Below are the rationales, notes, and caveats for this claim, organized by each r
+* PATCH version when you make backwards compatible bug fixes. PATCH is binary compatible.
+
+Additional labels for pre-release and build metadata are available as extensions to the MAJOR.MINOR.PATCH format.
+This is what has been the case and we see no reason not to follow this going forward.
+Thus far, there’s been no MINOR release that has broken binary compatibility. And note that this only holds for the stable interface. Along with bumping MAJOR to 1, this policy will be reconsidered and it may be decided to guarantee binary compatibility for MINOR versions is as well.
+
+The CMake sources configure the compatibility mode to [`SameMajorVersion`](https://cmake.org/cmake/help/latest/module/CMakePackageConfigHelpers.html#generating-a-package-version-file), shows that breaking changes are accompanied by a change to the `MAJOR` version part.
+Here is information in the [Releases section](https://www.eclipse.org/projects/handbook/#release) of the Eclipse project handbook which discusses _Major_, _Minor_, and _Service_ release criteria.
 
 ### Version Stability [1.ii]
 
-The latest release of `CycloneDDS` is considered stable, but there is no documentation about whether all future releases should be considered stable.
+`CycloneDDS` is at a stable version. The current version can be found in its [package.xml](https://github.com/eclipse-cyclonedds/cyclonedds/blob/master/package.xml), and its change history can be found in its [CHANGELOG](https://github.com/eclipse-cyclonedds/cyclonedds/blob/master/CHANGELOG.rst).
 
 ### Public API Declaration [1.iii]
 
-The public API for `CycloneDDS` is made up of symbols defined in the header files it installs.
-In the source repository, these header files reside with each of the modules that make up `CycloneDDS`.
+All symbols in the installed headers are considered part of the public API. In the source repository, these header files reside with each of the modules that make up CycloneDDS.
 
 ### API Stability Policy [1.iv]
 
-There is no published policy for API stability in `CycloneDDS`, however there is some information in the [Releases section](https://www.eclipse.org/projects/handbook/#release) of the Eclipse project handbook which states that releases which include breaking API changes are considered "Major".
+`CycloneDDS` will not break public API within a released ROS distribution, i.e. no major releases once the ROS distribution is released. It conforms to the [Releases section](https://www.eclipse.org/projects/handbook/#release) of the Eclipse project handbook which states that releases which include breaking API changes are considered "Major".
 
 ### ABI Stability Policy [1.v]
 
-There is no published policy for ABI stability in `CycloneDDS`.
+`CycloneDDS` provides ABI stability for PATCH release. `CycloneDDS` strives to provide ABI stability for MINOR release. `CycloneDDS` not guarantee ABI stability for MAJOR release.
```

Comment:
```suggestion
`CycloneDDS` provides ABI stability for PATCH releases. `CycloneDDS` strives to provide ABI stability for MINOR releases. MAJOR releases are unlikely to be ABI stable.
```

Is ABI stability checked somehow?

Reply:
@ThijsSassen @eboasson can you comment on @brawner's question?